src/test/transaction_tests.cpp

```diff
@@ -152,14 +152,16 @@ unsigned int FillFlags(unsigned int flags)
     return flags;
 }
 
-// Return valid flags that are all except one flag for each flag
-std::vector<unsigned int> ExcludeIndividualFlags(unsigned int flags)
+// Exclude each possible script verify flag from flags. Returns a list of these flag combinations
+// that are valid and without duplicates. For example: if flags=1111 and there are 4 possible flags,
+// this should return the vector [0111, 1011, 1101, 1110].
+std::set<unsigned int> ExcludeIndividualFlags(unsigned int flags)
 {
-    std::vector<unsigned int> flags_combos;
+    std::set<unsigned int> flags_combos;
     for (unsigned int i = 0; i < mapFlagNames.size(); ++i) {
```

Comment:
how is the map supposed to be kept up to date? Looks like it is missing SCRIPT_VERIFY_DISCOURAGE_UPGRADABLE_PUBKEYTYPE?

Reply:
Hm, is it possible to iterate through all the script verify flags? Could add an assert that they're all present in `mapFlagNames`

Reply:
Oh, my comment was just an example. There are more missing than just the one, IIRC

Reply:
should all be there now